Mathlib/Algebra/Group/Pointwise/Set/Basic.lean

```diff
@@ -3,12 +3,10 @@ Copyright (c) 2019 Johan Commelin. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin, Floris van Doorn, Yaël Dillies
 -/
-import Mathlib.Algebra.Group.Equiv.Basic
 import Mathlib.Algebra.Group.Prod
-import Mathlib.Algebra.Group.Units.Hom
-import Mathlib.Algebra.Opposites
 import Mathlib.Algebra.Order.Monoid.Unbundled.Pow
 import Mathlib.Data.Set.Lattice
+import Mathlib.Tactic.MinImports
```

Comment:
I tend to remove this import after I'm done with it, but am also happy to keep it around.

Reply:
I prefer to keep it in because IMO one should always have access to `#min_imports`. In fact, I would be happier if it were imported in `Mathlib.Init`